api/src/services/listing-csv-export.service.ts

```diff
@@ -431,6 +478,24 @@ export class ListingCsvExporterService implements CsvExporterServiceInterface {
         label: 'Home Type',
       });
     }
+
+    if (
+      this.doAnyJurisdictionHaveFeatureFlagSet(
+        user.jurisdictions,
+        FeatureFlagEnum.enableUnitGroups,
+      )
+    ) {
+      headers.push({
+        path: 'unitGroups.length',
+        label: 'Number of Unit Groups',
+      });
+    } else {
+      headers.push({
+        path: 'units.length',
+        label: 'Number of Units',
+      });
+    }
```

Comment:
Issue: same here. If there is a case where a unitGroup jurisdiction exists along with a unit jurisdiction we will want both columns to appear. I don't know if in production we'll ever see that, but we should handle it just in case